src/ecc.rs

```diff
@@ -21,6 +21,8 @@ use object::{
 use once_cell::sync::Lazy;
 use std::fmt::Debug;
 
+include! {"ecc-misc.rs"}
```

Comment:
why include this instead of simply importing via "use" ?

Reply:
I think initially I had there some macros or some other stuff that did not like the `use` but in the end it ended up quite slim which might work with normal use. Let me try to change that

Reply:
Updated to avoid the `include`.